frontend/src/pages/CheckEventPage.tsx

```diff
@@ -1,10 +1,22 @@
+import Timetable from '@/components/Timetable';
 import useCheckRoomSession from '@/hooks/useCheckRoomSession';
+import useUserAvailability from '@/hooks/useUserAvailablity';
 
 const CheckEventPage = () => {
   const roomInfo = useCheckRoomSession();
-  // ex) 사용시, roomInfo.title, room.availableDates
+  const userAvailability = useUserAvailability();
 
-  return <div>CheckEventPage</div>;
+  return (
+    <div>
+      <Timetable
+        startTime={roomInfo.time.startTime}
+        endTime={roomInfo.time.endTime}
```

Comment:
이것도 그냥 time으로 한 번에 넘기고 내부에서 전개연산자 쓰는 방향은 어떨까요?
```suggestion
        time={roomInfo.time}
```